mfr/extensions/tabular/render.py

```diff
@@ -1,5 +1,6 @@
 import os
 import json
+import _csv
```

Comment:
`_csv` is the internal C module that python's `csv` module wraps.  `csv` re-exports `_csv.Error` as its own `Error`. Use that instead, since that should have a stable interface.